docs/subnets/customize-a-subnet.md

```diff
@@ -152,6 +161,53 @@ The fields `nonce`, `timestamp`, `extraData`, `gasLimit`, `difficulty`, `mixHash
 set to match the `gasLimit` set in the `feeConfig`. You do not need to change any of the other genesis
 header fields.
 
+`nonce`, `mixHash` and `difficulty`
+
+These are remnant parameters from Proof of Work systems.
+For Avalanche, these don't play any relevant role, so you should just leave them as: 
+
+- `nonce` as `0x0`.
+- `mixHash` as `0x0000000000000000000000000000000000000000000000000000000000000000`.
+- `diffuculty` as `0x0.
```

Comment:
Why we skipped explaining these? I know they're not very significant but again none of these Header fields actually matter and they should be left as they're (in most cases). So I feel like if other header fields need explanation, these 3 fields also should be explained too.